styles.css

```diff
@@ -0,0 +1,115 @@
+body {
+  background-color: #00cdcd;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.nav {
+  overflow: hidden;
+}
+
+.nav-items {
+  list-style: none;
+  padding: 0;
+  float: right;
+  display: block;
+}
+
+.nav-item {
+  display: inline-block;
+}
+
+.nav-link {
+  padding: 20px;
+  color: #ffffff;
+  text-decoration: none;
+  display: block;
+}
+
+.nav-link:hover {
```

Comment:
e có thể viết là .nav-link:not(.dropdown-toggle):hover {
  border: 2px solid #ffffff;
}

thì sẽ ko cần phải reset lại border-bottom của thằng .dropdown-toggle về 0 ở dòng 36 nữa